experimental/traffic-portal/src/app/api/change-logs.service.ts

```diff
@@ -0,0 +1,40 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { HttpClient } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+
+import { APIService } from "src/app/api/base-api.service";
+import { ChangeLog } from "src/app/models/change-logs";
+
+/**
+ * Defines & handles api endpoints related to change logs
+ */
+@Injectable()
+export class ChangeLogsService extends APIService {
+
+	constructor(http: HttpClient) {
+		super(http);
+	}
+
+	/**
+	 * Calls api logs endpoint
+	 *
+	 * @param params Request paramteres to add
+	 * @returns Change logs
+	 */
+	public async getChangeLogs(params: Record<string, string>): Promise<Array<ChangeLog>> {
+		return this.get<[ChangeLog]>("logs", undefined, params).toPromise();
```

Comment:
This is returning a tuple of one ChangeLog item, not an array. It's working because `[T]` is assignable to `T[]`, but it'd be more accurate if they were the same type.